### DISCUSSION TITLE
Can this be implemented in a telegram bot?

### DISCUSSION THREAD

**Discussion by ronakborkar** [lllyasviel/Fooocus#2857] [Ideas]
It would be so awesome if someone could develop this in a way that could work as a telegram bot with subscription or token based model

**Comment by mashb1t** [comment]
Fooocus relies on Gradio as frontend framework and in its current form isn't able to be run on CLI only. If somebody would make this possible, certainly then Fooocus could be turned into a bot for various use cases, but until then it's more convenient to use another tool for this purpose.
